packages/core/src/tools/request-merge.js

```diff
@@ -13,6 +14,9 @@ const caseInsensitiveMerge = (requestOne, requestTwo, requestThree) => {
 
   // This is a very quick & efficient merge for all of request's properties
   const mergedRequest = _.merge(requestOne, requestTwo, requestThree);
+  if (requestThree[REPLACE_CURLIES]) {
+    mergedRequest[REPLACE_CURLIES] = requestThree[REPLACE_CURLIES];
+  }
```

Comment:
Is this because `_.merge` doesn't handle symbols well? If so, let's add a comment to explain.

Reply:
Added in [c89bd2c](https://github.com/zapier/zapier-platform/pull/1032/commits/c89bd2c39f59000ab070c23cc9c2ab9b970c15e6).